src/content/en/updates/2019/07/image-support-for-async-clipboard.md

```diff
@@ -203,8 +210,8 @@ async function getClipboardContents() {
 
 ### Custom paste handler {: #custom-paste-handler }
 
-If you want to dynamically handle paste events, you can listen for the `paste`
-event, [prevent the default][prevent-default] behavior, then use the code above
+To dynamically handle paste events, listen for the `paste`
+event, call [`preventDefault()`][prevent-default], then use the code above
```

Comment:
Why, specifically, is `preventDefault()` being called? I'm not able to infer it. It might be something we need to explain.

Reply:
Oh, simply because you might want to do anything else before actually pasting, like maybe crop to square and zoom on face if you work on something like an avatar picker or so.

Reply:
That is certainly worth mentioning in the text, and not something I'd assume readers would just think of.

Reply:
Added a clarification.